### PR TITLE
Write an ADR for the ES 5 upgrade

### PR DESCRIPTION
**Pull request by barrucadu**
[Trello card](https://trello.com/c/5zsSYGzq/661-write-adrs-for-last-quarters-work?menu=filter&filter=label:Finders,label:Search)

### CODE REVIEW

**Review by brucebolt — APPROVED** [review]
Great work, the level of detail is brilliant 👍

**Comment by barrucadu** [comment]
:+1: I'll get someone from govuk-finding-brexit to make sure it makes sense to people other than us.

File: doc/arch/adr-008-elasticsearch5-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+- **Very strange result ordering.** We resolved this by switching back
+  to the classic text similarity and back to using `string` fields.
+  These are mentioned in more detail further up.
+
+- **Every query returned every best bet.** This was due to a mistake
+  made when combining all the types.
+
+- **Indices would automatically unlock.** This was due to an AWS
+  process which runs every four minutes: locking the indices if the
+  cluster is unlhealthy and unlocking them if it is healthy.  We
```
Comment (erino) [inline doc/arch/adr-008-elasticsearch5-upgrade.md]: Typo in `unlhealthy`

**Review by erino — CHANGES_REQUESTED** [review]
Just a small typo

**Review by erino — APPROVED** [review]

**Pull request merged**